{"protocolVersion":"0.1.1","runnerVersion":null,"pid":14558,"type":"start","time":0}
{"suite":{"id":0,"platform":"vm","path":"/Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart"},"type":"suite","time":0}
{"test":{"id":1,"name":"loading /Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart","suiteID":0,"groupIDs":[],"metadata":{"skip":false,"skipReason":null},"line":null,"column":null,"url":null},"type":"testStart","time":0}
{"count":1,"time":2,"type":"allSuites"}

[{"event":"test.startedProcess","params":{"vmServiceUri":"http://127.0.0.1:54955/dHSFvsDEHeo=/","observatoryUri":"http://127.0.0.1:54955/dHSFvsDEHeo=/"}}]
{"testID":1,"result":"success","skipped":false,"hidden":true,"type":"testDone","time":1586}
{"group":{"id":2,"suiteID":0,"parentID":null,"name":"","metadata":{"skip":false,"skipReason":null},"testCount":1,"line":null,"column":null,"url":null},"type":"group","time":1587}
{"test":{"id":3,"name":"Counter increments smoke test","suiteID":0,"groupIDs":[2],"metadata":{"skip":false,"skipReason":null},"line":153,"column":5,"url":"package:flutter_test/src/widget_tester.dart","root_line":15,"root_column":3,"root_url":"file:///Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart"},"type":"testStart","time":1587}
{"testID":3,"messageType":"print","message":"══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════\nThe following assertion was thrown building Scaffold(dirty, state: ScaffoldState#ce771(tickers:\ntracking 2 tickers)):\nNo Directionality widget found.\nScaffold widgets require a Directionality widget ancestor.\nThe specific widget that could not find a Directionality ancestor was:\n  Scaffold\nThe ownership chain for the affected widget is: \"Scaffold ← MyHomePage ← MediaQuery ←\n  _MediaQueryFromView ← _ViewScope ← View-[GlobalObjectKey TestFlutterView#1c987] ← [root]\"\nTypically, the Directionality widget is introduced by the MaterialApp or WidgetsApp widget at the\ntop of your application widget tree. It determines the ambient reading direction and is used, for\nexample, to determine how to lay out text, how to interpret \"start\" and \"end\" values, and to resolve\nEdgeInsetsDirectional, AlignmentDirectional, and other *Directional objects.\n\nThe relevant error-causing widget was:\n  Scaffold\n  Scaffold:file:///Users/hyperlink/StudioProjects/custom_map/lib/home_page/my_home_page.dart:58:12\n\nWhen the exception was thrown, this was the stack:\n#0      debugCheckHasDirectionality.<anonymous closure> (package:flutter/src/widgets/debug.dart:353:7)\n#1      debugCheckHasDirectionality (package:flutter/src/widgets/debug.dart:373:4)\n#2      ScaffoldState.build (package:flutter/src/material/scaffold.dart:2731:12)\n#3      StatefulElement.build (package:flutter/src/widgets/framework.dart:5198:27)\n#4      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5086:15)\n#5      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)\n#6      Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#7      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:5068:5)\n#8      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5242:11)\n#9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:5062:5)\n...     Normal element mounting (9 frames)\n#18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3971:16)\n#19     Element.updateChild (package:flutter/src/widgets/framework.dart:3702:20)\n#20     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)\n#21     Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#22     ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)\n#23     Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)\n#24     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)\n#25     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5251:11)\n#26     Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#27     StatefulElement.update (package:flutter/src/widgets/framework.dart:5274:5)\n#28     Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)\n#29     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)\n#30     Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#31     ProxyElement.update (package:flutter/src/widgets/framework.dart:5417:5)\n#32     Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)\n#33     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:5111:16)\n#34     Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#35     StatelessElement.update (package:flutter/src/widgets/framework.dart:5162:5)\n#36     Element.updateChild (package:flutter/src/widgets/framework.dart:3686:15)\n#37     RenderObjectToWidgetElement._rebuild (package:flutter/src/widgets/binding.dart:1240:16)\n#38     RenderObjectToWidgetElement.update (package:flutter/src/widgets/binding.dart:1217:5)\n#39     RenderObjectToWidgetElement.performRebuild (package:flutter/src/widgets/binding.dart:1231:7)\n#40     Element.rebuild (package:flutter/src/widgets/framework.dart:4805:7)\n#41     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2780:19)\n#42     AutomatedTestWidgetsFlutterBinding.drawFrame (package:flutter_test/src/binding.dart:1386:19)\n#43     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:358:5)\n#44     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1284:15)\n#45     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1214:9)\n#46     AutomatedTestWidgetsFlutterBinding.pump.<anonymous closure> (package:flutter_test/src/binding.dart:1236:9)\n#49     TestAsyncUtils.guard (package:flutter_test/src/test_async_utils.dart:68:41)\n#50     AutomatedTestWidgetsFlutterBinding.pump (package:flutter_test/src/binding.dart:1222:27)\n#51     WidgetTester._pumpWidget (package:flutter_test/src/widget_tester.dart:592:20)\n#52     WidgetTester.pumpWidget.<anonymous closure> (package:flutter_test/src/widget_tester.dart:577:14)\n#55     TestAsyncUtils.guard (package:flutter_test/src/test_async_utils.dart:68:41)\n#56     WidgetTester.pumpWidget (package:flutter_test/src/widget_tester.dart:576:27)\n#57     main.<anonymous closure> (file:///Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart:17:18)\n#58     testWidgets.<anonymous closure>.<anonymous closure> (package:flutter_test/src/widget_tester.dart:170:29)\n<asynchronous suspension>\n<asynchronous suspension>\n(elided 5 frames from dart:async and package:stack_trace)\n\n════════════════════════════════════════════════════════════════════════════════════════════════════","type":"print","time":1732}
{"testID":3,"messageType":"print","message":"══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════\nThe following TestFailure was thrown running a test:\nExpected: exactly one matching node in the widget tree\n  Actual: _TextFinder:<zero widgets with text \"0\" (ignoring offstage widgets)>\n   Which: means none were found but one was expected\n\nWhen the exception was thrown, this was the stack:\n#4      main.<anonymous closure> (file:///Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart:20:5)\n<asynchronous suspension>\n<asynchronous suspension>\n(elided one frame from package:stack_trace)\n\nThis was caught by the test expectation on the following line:\n  file:///Users/hyperlink/StudioProjects/custom_map/test/widget_test.dart line 20\nThe test description was:\n  Counter increments smoke test\n════════════════════════════════════════════════════════════════════════════════════════════════════","type":"print","time":1735}
{"testID":3,"messageType":"print","message":"══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════\nThe following message was thrown:\nMultiple exceptions (2) were detected during the running of the current test, and at least one was\nunexpected.\n════════════════════════════════════════════════════════════════════════════════════════════════════","type":"print","time":1735}
{"testID":3,"error":"Test failed. See exception logs above.\nThe test description was: Counter increments smoke test","stackTrace":"","isFailure":false,"type":"error","time":1736}
{"testID":3,"result":"error","skipped":false,"hidden":false,"type":"testDone","time":1737}
{"success":false,"type":"done","time":2003}
